src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -208,19 +212,11 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
     const std::string element_name =
         MakeString{} << ElementId<VolumeDim>(array_index) << '/';
 
-    // calculate the interpolation matrices if requested. Else, interp_matrices
-    // will not be default constructed and therefore skipped by applyMatrices.
-    std::array<Matrix, VolumeDim> interp_matrices{};
-    if (interpolation_mesh_) {
-      const auto slices = mesh.slices();
-      const auto interpolation_slices = interpolation_mesh_->slices();
-      for (size_t i = 0; i < VolumeDim; i++) {
-        const auto target_points =
-            Spectral::collocation_points(interpolation_slices.at(i));
-        interp_matrices.at(i) =
-            Spectral::interpolation_matrix(slices.at(i), target_points);
-      }
-    }
+    // if no interpolation_mesh is provided, the interpolation is essentially
+    // ignored by the RegularGridInterpolant
```

Comment:
..except for a copy. Probably irrelevant, since the non-solution tensors have to be copied anyway and optimizing this just for the errors seems premature. Please add a note to the comment though.